app/javascript/packages/password-confirmation/package.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "@18f/identity-password-confirmation",
+  "private": true,
+  "version": "1.0.0",
+  "peerDependencies": {
+    "react": "^17.0.2"
+  }
```

Copying comment from https://github.com/18F/identity-idp/pull/8161#discussion_r1163290254 :

>It doesn't appear that we're using React in the package.

```suggestion
  "version": "1.0.0"
```